packages/core/src/electron-browser/window/electron-window-preferences.ts

```diff
@@ -38,11 +39,24 @@ export const electronWindowPreferencesSchema: PreferenceSchema = {
             // eslint-disable-next-line max-len
             'description': 'Adjust the zoom level of the window. The original size is 0 and each increment above (e.g. 1.0) or below (e.g. -1.0) represents zooming 20% larger or smaller. You can also enter decimals to adjust the zoom level with a finer granularity.'
         },
+        'window.titleBarStyle': {
+            type: 'string',
+            enum: ['native', 'custom'],
+            markdownEnumDescriptions: [
+                'Native title bar is displayed.',
+                'Custom title bar is displayed.'
+            ],
+            default: isWindows ? 'custom' : 'native',
+            scope: 'application',
+            markdownDescription: 'Adjust the appearance of the window title bar. Changes require a full restart to apply',
```

Comment:
We might want to align here, and reference context-menus which it also updates:

```suggestion
            markdownDescription: 'Adjust the appearance of the window title bar. On Linux and Windows, this setting also affects the application and context menu appearances. Changes require a full restart to apply.',
```